axum-extra/src/routing/spa.rs

```diff
@@ -153,18 +167,40 @@ where
     HandleError<Route<B, io::Error>, F, T>:
         Service<Request<B>, Response = Response, Error = Infallible>,
     <HandleError<Route<B, io::Error>, F, T> as Service<Request<B>>>::Future: Send,
-    B: HttpBody + Send + 'static,
+    B: HttpBody + Default + Send + 'static,
```

Comment:
This makes it a breaking change. Don't think there is a way around it since we need to first call the assets service and if that 404s then call the index service.

Reply:
Didn't you want to publish a breaking release of axum-extra soon anyways? :)

Reply:
yep I do :)

Reply:
Oh this thread was 3 days old.. I think I arrived here from a notification though? Weird 😄